laravel.md

```diff
@@ -0,0 +1,52 @@
+# Laravel
+
+**Prerequisite knowledge:** php, oop, mysql, html, frameworks, mvc, composer, dependency managment
```

Comment:
I've learned oop, mvc, frameworks and composer as I was making my first project in Symfony 2. And it seems natural to me that you explore those concepts as you explore your first MVC framework. Therefore I'm not sure they should be classified as prerequisite knowledge... It's rather a digress than suggestion though. 😄

Reply:
I think it depends on the sources that you use. Many books that are introducing you to a framework starts with explaining many of these areas, but I saw some articles that skips it. I think it's better to point it in two places than skipping.

Reply:
I think our dependencies system isn't really about "prerequisite knowledge" but more about the things you'll also learn in the process.

Reply:
We can start a discussion in a separate issue or PR.